.github/workflows/prod.yml

```diff
@@ -0,0 +1,58 @@
+name: Production
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  laravel-tests:
+    runs-on: self-hosted
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Copy .env
+      run: php -r "file_exists('.env') || copy('.env.example', '.env');"
+      working-directory: ./src
+    - name: Install Dependencies
+      run: composer install -q --no-ansi --no-interaction --no-scripts --no-suggest --no-progress --prefer-dist
+      working-directory: ./src
+    - name: Generate key
+      run: php artisan key:generate
+      working-directory: ./src
+    - name: Directory Permissions
+      run: chmod -R 777 storage bootstrap/cache
+      working-directory: ./src
+    - name: Create Database
+      run: |
+        mkdir -p database
+        touch database/database.sqlite
```

Comment:
why sqlite? is the worst sql motor existing. And we actualy use postgress for dev why not use it on prod?

Reply:
Fixed in 2b736399840ba074fc05ec056276a0853684d7ab ae011bb9f04794b06fe1a6e4a5a02cff7a0a2b1a

Reply:
This doesn't seem to be fixed.